x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select.tsx

```diff
@@ -39,44 +233,51 @@ export const ModelSelect: React.FC = () => {
     }));
 
   const onChange = (options: ModelSelectOptionProps[]) => {
-    const selectedOption = options.find((option) => option.checked === 'on');
+    const selectedModelOption = options.find((option) => option.checked === 'on');
+
     setInferencePipelineConfiguration({
       ...configuration,
       inferenceConfig: undefined,
-      modelID: selectedOption?.modelId ?? '',
-      isModelPlaceholderSelected: selectedOption?.isPlaceholder ?? false,
+      modelID: selectedModelOption?.modelId ?? '',
+      isModelPlaceholderSelected: selectedModelOption?.isPlaceholder ?? false,
       fieldMappings: undefined,
       pipelineName: isPipelineNameUserSupplied
         ? pipelineName
-        : indexName + '-' + normalizeModelName(selectedOption?.modelId ?? ''),
+        : indexName + '-' + normalizeModelName(selectedModelOption?.modelId ?? ''),
     });
   };
 
   const renderOption = (option: ModelSelectOptionProps) => <ModelSelectOption {...option} />;
 
   return (
-    <EuiSelectable
-      data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-configureInferencePipeline-selectTrainedModel`}
-      options={getModelSelectOptionProps(selectableModels)}
-      singleSelection="always"
-      listProps={{
-        bordered: true,
-        rowHeight: useIsWithinMaxBreakpoint('s') ? 180 : 90,
-        showIcons: false,
-        onFocusBadge: false,
-      }}
-      height={360}
-      onChange={onChange}
-      renderOption={renderOption}
-      isLoading={isLoading}
-      searchable
-    >
-      {(list, search) => (
-        <>
-          {search}
-          {list}
-        </>
-      )}
-    </EuiSelectable>
+    <EuiFlexGroup>
+      <EuiFlexItem>
+        <EuiSelectable
+          data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-configureInferencePipeline-selectTrainedModel`}
+          options={getModelSelectOptionProps(selectableModels)}
+          singleSelection="always"
+          listProps={{
+            bordered: true,
+            rowHeight: useIsWithinMaxBreakpoint('s') ? 128 : 96,
+            onFocusBadge: false,
+          }}
+          height={384}
```

Comment:
Height is n*32px as per EUI [sizing guidelines](https://eui.elastic.co/#/theming/sizing/values).

Reply:
Updated logic.